src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeKernelMap.cs

```diff
@@ -167,12 +184,14 @@ public void Dispose()
             return result;
         }
 
-        protected virtual void Initialize()
+        /// <summary>
+        /// Initializes the kernel map.
+        /// </summary>
+        protected internal virtual void Initialize()
```

Comment:
If we could pass `TResampler` as an argument to this method instead of storing it as a member, we could flip back `ResizeKernelMap` to a non-generic class, with only the methods `Calculate<TResampler>(..)` and `Initialize<TResampler>(...) being generic.

Reply:
Ah that would be good yes.